README.md

```diff
@@ -19,15 +19,15 @@ For understanding the overall objective of the library please refer to [openmine
 
 - [x] Create the basic structure of KotlinSyft
 - [x] Implement WebRTC flow in Android
-- [ ] Implement Protobuf in Android
+- [x] Implement Protobuf in Android
 - [x] Set up deployment to Maven
 - [ ] Implement split and stitch algorithm for data channels in Android
 - [x] Add support for background task scheduling in Android
 - [ ] Implement sleep/wake detection in Android
```

Comment:
I think we should remove these. They require a commit specifically for giving updates